### PR TITLE
Fix shimmer card component in mine page

### LINKED ISSUE #1283: Fix: change shimmer ui card in mine page
### Issue Description

The shimmer card component is not consistent with actual task card which makes it non user friendly.

### Expected Behavior

https://github.com/user-attachments/assets/d397b278-997b-4b0b-a63c-d92628e28e28

### Current Behavior
[screen-capture (3).webm](https://github.com/user-attachments/assets/114fd4d3-bfd1-4be0-8400-adb027e4b2ba)

### Reproducibility

 - [x] This issue is reproducible
 - [ ] This issue is not reproducible

### Steps to Reproduce
1. Go to [main page](https://status.realdevsquad.com/mine)

### Severity/Priority
 - [ ] Critical
 - [ ] High
 - [ ] Medium
 - [x] Low

### Checklist

- [x] I have read and followed the project's code of conduct.
- [x] I have searched for similar issues before creating this one.
- [x] I have provided all the necessary information to understand and reproduce the issue.
- [x] I am willing to contribute to the resolution of this issue.

### PR DESCRIPTION
**Pull request by pankajjs**
Date: 20 Nov, 2024

Developer Name: @pankajjs 

---

## Issue Ticket Number

- Closes #1283 

## Description
This PR fixes the shimmer card component in mine page.

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

https://github.com/user-attachments/assets/4953fd97-e9a4-470d-aeab-e8f148738538

Screenshot 2024-11-25 at 11 27 42 PM
Screenshot 2024-11-25 at 11 27 53 PM

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>
Screenshot 2024-11-20 at 1 03 29 AM

</details>

## Additional Information

Design Doc - https://docs.google.com/document/d/1Y-V7ClFjge1c01ErrFpGgmfZJ6wXZCJ7q2SuLDrcEcY/edit?tab=t.0#heading=h.iwhskjpqyzwf

### CODE REVIEW

File: src/pages/mine/index.tsx [hunk 1]
```diff
@@ -57,9 +57,9 @@ const Content = () => {
 
     if (isLoading)
         return (
-            <div className={styles.tasksContainer}>
+            <div className={styles.mineTasksContainer}>
```
Comment (VinuB-Dev) [inline src/pages/mine/index.tsx:60]: Did we already have this class? I can't find it defined in the new code.

Comment (pankajjs) [reply src/pages/mine/index.tsx:60]: it is already there.

**Comment by VinuB-Dev** [comment]
Code looks good.

**Comment by VinuB-Dev** [comment]
@pankajjs Can you also add mobile view recording / screenshot.

**Comment by pankajjs** [comment]
@VinuB-Dev Have you checked the screenshots? If yes, could you approve it?

**Review by VinuB-Dev — APPROVED** [review]
Code looks good.

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**